tests/sources/fixtures/github/fixture.py

```diff
@@ -31,158 +28,269 @@ def encode_data(content):
 class GitHubAPI:
     def __init__(self):
         self.app = Flask(__name__)
-        self.pull_request_pages = "15"
-        self.issue_pages = "15"
-        self.file_count = 2500
-        self.app.route("/user", methods=["GET"])(self.get_user)
-        self.app.route("/user/repos", methods=["GET"])(self.get_repos)
-        self.app.route("/repos/demo_user/demo_repo/pulls", methods=["GET"])(
-            self.get_pull_requests
-        )
-        self.app.route(
-            "/repos/demo_user/demo_repo/issues/<string:id>/comments", methods=["GET"]
-        )(self.get_comments)
-        self.app.route(
-            "/repos/demo_user/demo_repo/pulls/<string:id>/comments", methods=["GET"]
-        )(self.get_comments)
+        self.file_count = 3500
+        self.issue_count = 1000
```

Comment:
Should `file_count` and `issue_count` also depend on `_SIZES`?

Reply:
No, actually count is same for all the data sizes. we are changing the content of the files only for the different sizes.